product/roundhouse/runners/RoundhouseMigrationRunner.cs

```diff
@@ -39,6 +40,7 @@ public sealed class RoundhouseMigrationRunner : IRunner
             bool silent,
             bool dropping_the_database,
             bool dont_create_the_database,
+            bool dont_alter_the_database,

```

Comment:
I'm a little lost on this since the configuration has the value, why do you need to add this?